src/fields/IntegerField.cpp

```diff
@@ -1,16 +1,16 @@
 #include "IntegerField.h"
 
-IntegerField::IntegerField(string name, int defaultValue, bool primary, bool unique, bool nullable, bool autoInc): 
-    Field(name, INTEGER, primary, unique, nullable, autoInc) { 
-    fieldValue = defaultValue;
-    autoIncrement = autoInc;
+IntegerField::IntegerField(string fieldName, int defaultValue, bool primaryKey, bool unique, bool nullable, bool autoIncrement): 
+    Field(fieldName, INTEGER, primaryKey, unique, nullable, autoIncrement) { 
+    _fieldValue = defaultValue;
+    _autoIncrement = autoIncrement; //Should this be passed to Field (above)?

```

Comment:
Just checking. My thinking is that:
1) We're creating a new variable _autoIncrement for the IntegerField class when 
2) The Field class already has a _autoFilled variable (which autoIncrement is passed to).
... which seemed a bit weird. Will remove the comment, though.